File: WeatherServer/WeathARServer/src/main/java/de/htw/mm/weathar/model/WeatherForecast.java
package de.htw.mm.weathar.model;

public class WeatherForecast {

    private WeatherReport today;

    private WeatherReport tomorrow;

    private WeatherReport dayAfterTomorrow;

    public WeatherReport getToday() {
        return today;
    }

    public void setToday(WeatherReport today) {
        this.today = today;
    }

    public WeatherReport getTomorrow() {
        return tomorrow;
    }

    public void setTomorrow(WeatherReport tomorrow) {
        this.tomorrow = tomorrow;
    }

    public WeatherReport getDayAfterTomorrow() {
        return dayAfterTomorrow;
    }

    public void setDayAfterTomorrow(WeatherReport dayAfterTomorrow) {
        this.dayAfterTomorrow = dayAfterTomorrow;
    }
}

File: WeathAR/Assets/Scripts/Model/WeatherForecast.cs
using System;

namespace Model
{
    /**
     * Representation of the weather forecase construct. Is used as a wrapper object for the response json.
     *
     * @author Cong Minh Nguyen, Tuan Tung Tran
     * @date 20.07.2019
     * 
     */
    [Serializable]
    public class WeatherForecast
    {
        public WeatherReport today { set; get; }

        public WeatherReport tomorrow { set; get; }

        public WeatherReport dayAfterTomorrow { set; get; }
    }
}